exercises_on_code_quality/ReverseVowelsInAString.js

```diff
@@ -0,0 +1,10 @@
+const reverseVowels = (str) => {
+  //The regex help to test or match all vowels in the string with out considering upper or lower cases.
+  let vowels = str.match(/[aiuoe]/gi);
```

Comment:
I would suggest you to change **let** vowels into **const** vowels because it's not being reassigned